src/rider/main/kotlin/me/seclerp/rider/plugins/efcore/features/migrations/add/AddMigrationDialogWrapper.kt

```diff
@@ -67,6 +70,29 @@ class AddMigrationDialogWrapper(
         }
     }
 
+    override fun Panel.createAdditionalGroup() {
+        groupRowsRange("Additional Options"){
+            row("Migrations folder:") {
+                val fileChooserDescriptor =  FileChooserDescriptor(
+                    /* chooseFiles = */ false,
+                    /* chooseFolders = */ true,
+                    /* chooseJars = */ false,
+                    /* chooseJarsAsFiles = */ false,
+                    /* chooseJarContents = */ false,
+                    /* chooseMultiple = */ false
+                )
+
+                textFieldWithBrowseButton("Select Migrations Folder", intellijProject, fileChooserDescriptor) {
+                    val migrationsFolder : VirtualFile = it
+                    formatMigrationFolderPath(migrationsFolder)
```

Comment:
Please encapsulate this logic into a function inside `DslExtensions`. Something like `textFieldForRelativeFolder`